docs/guides/BasicConversation.md

```
@@ -1,57 +1,79 @@
-    identifier: dd2a86730a3441939359e960f0cc2da3
-    role: model
-    author: None
-    created: 2025-03-07 12:40:30.130777+00:00
-    content:
-      parts:
-        - text: The current UTC time and date is Friday, 7th March 2025, 12:40:29.
-          meta: None
-    meta: None
+Example output (truncated for clarity):
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)